README.md

```diff
@@ -39,6 +39,10 @@ $ npm install
 # service worker is disabled in dev
 $ npm run dev
 
+# If port 3000 is already taken, you can specify a port, e.g. port 3001.
+# Assumes you installed Nuxt in the project folder.
+node_modules/.bin/nuxt -p 3001
```

Comment:
i this case you can also update your package.json like that
```js
  "scripts": {
    "dev": "PORT=5000 node_modules/.bin/nuxt",
```
The idea is that most of the time, you just want to run "npm run dev" or "npm run start" on all your projects, without asking yourself which is the binary or the exact command to launch the server

Reply:
( but it also forces people working with you to use the same port, not sure about the best practice here )